tutorials/advanced-development/01-advanced-pieces-topics/06-next-previous-links.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Using the Pieces Index"
+layout: tutorial
+---
+
+# Using the Pieces Index
+
+There are several options that you should know about for using pieces: creating "next" and "previous" links, fetching pieces programmatically, and removing pieces from a listing.
```

Comment:
This doesn't really make sense as a catch-all, it's not a general page about "using the pieces index."

"Fetching pieces programmatically" as a general topic is not really in this document, the section you're referring to is specifically about fetching the "next" and "previous" pieces programmatically. And "removing pieces from a listing" is unrelated to the rest of what you have in this page so you should move it to a separate page. I would suggest "suppressing" rather than "removing".